release/release.go

```diff
@@ -45,8 +45,7 @@ func (os *OS) ForceDevMode() bool {
 		fallthrough
 	case "ubuntu":
 		return false
-
-	case "elementary":
```

Comment:
Actually, even better: strings.Fields(s)[0].  Let's pick just the first token there.